### PR TITLE
Add info about supported currencies in payment gateways

### PR DESCRIPTION
**Pull request by IKarbowiak**
Add information about supported currencies in gateways.

### CODE REVIEW

**Comment by IKarbowiak** [comment]
@NyanKiyoshi where exactly do you want to add this? The `Payment gateway` page is about adding new gateways, there is no info about the existing ones.

File: docs/developer/checkout.mdx [hunk 1]
```diff
@@ -38,7 +38,7 @@ The resulting `Checkout` object contains the following fields:
 - `token`: similar to `id`, a unique identifier suitable for inclusion in emails and URLs.
 - `totalPrice`: the total price of the checkout lines and shipping costs.
 - `isShippingRequired`: denotes whether shipping is required for this checkout.
-- `availablePaymentGateways`: a list of payment gateways that are currently configured on your Saleor server and can be used to pay for the checkout. For each gateway, API returns a ID, name and a config object, which for some gateways may return additional information required to process the payment in the frontend.
+- `availablePaymentGateways`: a list of payment gateways that are currently configured on your Saleor server and can be used to pay for the checkout. Only gateways which support checkout currency are returned. For each gateway, API returns a ID, name, and a config object, which for some gateways may return additional information required to process the payment in the frontend.
```
Comment (patrys) [inline docs/developer/checkout.mdx:41]:
```suggestion
- `availablePaymentGateways`: a list of payment gateways that are currently configured on your Saleor server and can be used to pay for the checkout. Only gateways which support the checkout currency are returned. For each gateway, API returns an ID, a name, and a config object, which for some gateways may return additional information required to process the payment in the frontend.
```

File: docs/developer/checkout.mdx [hunk 2]
```diff
@@ -222,7 +222,7 @@ Depending on the selected payment gateway, you will either use the JavaScript fo
 The `checkoutPaymentCreate` mutation requires the following input:
 
 - `checkoutId`: the checkout ID.
-- `gateway`: the ID of the selected payment gateway (list of the available payment gateways can be fetched from the `Checkout.availablePaymentGateways` field).
+- `gateway`: the ID of the selected payment gateway (list of the available payment gateways can be fetched from the `Checkout.availablePaymentGateways` field). The selected gateway must support checkout currency.
```
Comment (patrys) [inline docs/developer/checkout.mdx:225]:
```suggestion
- `gateway`: the ID of the selected payment gateway (list of the available payment gateways can be fetched from the `Checkout.availablePaymentGateways` field). The selected gateway must support the checkout currency.
```

File: docs/developer/extending/payment-gateways.mdx [hunk 3]
```diff
@@ -8,7 +8,11 @@ Saleor uses a universal flow that each gateway should implement by providing a c
 All of the plugins method takes `previous_value` as a second argument.
 More information about `previous_value` can be found [here](plugins#base-plugin-class).
 
-For more information on the plugin implementation methods, see the `plugin.py` file for an existing payment gateway.
+For gateway to work correctly you also need to specified `Supported currencies` in gateway configuration.
```
Comment (patrys) [inline docs/developer/extending/payment-gateways.mdx:11]:
```suggestion
For a gateway to work correctly, you also need to specify `supported_currencies` in your gateway configuration.
```

File: docs/developer/extending/payment-gateways.mdx [hunk 4]
```diff
@@ -81,13 +85,14 @@ For more information on how to configure your plugin, see the [Plugins](dashboar
...
-| `connection_params` | `dict` | List of parameters used for connecting to the payment’s gateway                                          |
-| `store_customer`    | `bool` | If set to True, the system will save this payment method for this customer                               |
-| `require_3d_secure` | `bool` | Determines if gateway should enforce 3D secure verification during payment                               |
+| Name                   | Type   | Description                                                                                              |
+| ---------------------- | ------ | -------------------------------------------------------------------------------------------------------- |
+| `gateway_name`         | `str`  | Define gateway name                                                                                      |
+| `auto_capture`         | `bool` | Define if a gateway should also capture funds from the card. If false, payment should be only authorized |
+| `supported_currencies` | `str`  | Define currencies which are supported by gateway                                                         |
```
Comment (patrys) [inline docs/developer/extending/payment-gateways.mdx:92]:
```suggestion
| `supported_currencies` | `str`  | Define currencies which are supported by the gateway                                                     |
```

**Pull request merged**